scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -1261,17 +1262,6 @@ class SpelExpressionSpec extends AnyFunSuite with Matchers with ValidatedValuesD
             Some(CoordinatesBasedTextRange(TextCoordinates(start, 0), TextCoordinates(end, 0)))
           ) :: Nil if start == "ala #{".length && end == "ala #{.".length =>
     }
-
-    parse[String](
```

Comment:
Why did you remove it? Now SpEL handles this?

Reply:
The same error as above (`Unexpectedly ran out of input`) is returned.